Update secure-by-default.md (#6547)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/secure-by-default.md

- IP Allow List (connection filtering)
- Exchange mail flow rules (also known as transport rules)

> [!NOTE]

> > In case, a third party spam filter exists or MX record is not point to EOP, tenant overrides are applied.

Reviewer: ```suggestion
> If a third-party spam filter exists or the MX record does not point to EOP, tenant overrides are applied.
```
